src/Pages/Homepage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import fbIcon from '../static/images/icons8-facebook-48.png';
+import twitterIcon from '../static/images/icons8-twitter-48.png';
+import '../static/css/homepage.css';
+
+
+function Homepage() {
+  return (
+    <>
```

Comment:
empty element

Reply:
I tried using React.Fragment here to avoid adding random divs, but it seems the linter doesn't like that and replaces it with an empty tag. Is adding a div around each component generally okay? I thought it was kinda unclean but that's just my opinion

Reply:
It's what react likes, there are ways to avoid it but require more complicated structures. Additional divs around everything are very unexpensive, and you'll find that they often/easily become useful for more complex styling!